modules/cluster-management/src/main/java/org/apache/ignite/internal/cluster/management/raft/RaftStorageManager.java

```diff
@@ -42,16 +47,20 @@ class RaftStorageManager {
     /** Storage key for the CMG state. */
     private static final byte[] CMG_STATE_KEY = "cmg_state".getBytes(UTF_8);
 
-    /** Prefix for the keys for logical topology nodes. */
-    private static final byte[] LOGICAL_TOPOLOGY_PREFIX = "logical_".getBytes(UTF_8);
+    /** Storage key for the logical topology. */
+    private static final byte[] LOGICAL_TOPOLOGY_KEY = "logical".getBytes(UTF_8);
 
     /** Prefix for validation tokens. */
     private static final byte[] VALIDATED_NODE_PREFIX = "validation_".getBytes(UTF_8);
 
     private final ClusterStateStorage storage;
 
+    private volatile LogicalTopology currentLogicalTopology;
```

Comment:
maybe pre-set it with INITIAL?

Reply:
We initialize it in the constructor, so this seems excessive